pkg/synchromanager/clustersynchro_manager.go

```diff
@@ -341,6 +362,71 @@ func (manager *Manager) UpdateClusterStatus(ctx context.Context, name string, st
 	})
 }
 
+func (manager *Manager) addOrDeleteClusterSyncTemplate(obj interface{}) {
+	template := obj.(*clusterv1alpha2.ClusterSyncResourcesTemplate)
+	clusters, err := manager.clusterlister.List(labels.Everything())
+	if err != nil {
+		klog.Error("list clusters failed while updating template : %v", err)
+		return
+	}
+	for _, cluster := range clusters {
+		if cluster.Spec.SyncResourcesTemplate == template.Name {
+			manager.enqueue(cluster)
+			return
```

Comment:
Need to queue all clusters that reference the template

Reply:
done